flexmeasures/data/models/time_series.py

```diff
@@ -302,7 +302,7 @@ def __init__(
     @classmethod
     def search(
         cls,
-        sensor: Union[Sensor, int],
+        sensor: Union[Sensor, int, str, List[Union[Sensor, int, str]]],
```

Comment:
I'd rename this argument to `sensors`, so it communicates its capabilities. It also fits the docstring then.